api/src/infrastructure/docker.rs

```diff
@@ -884,13 +1026,41 @@ impl Infrastructure for DockerInfrastructure {
 
                 match status {
                     ServiceStatus::Running => {
-                        if !details.state.running {
-                            run_future_and_map_err!(c.start(), "Could not start container: {}");
+                        if !details
+                            .state
+                            .as_ref()
+                            .map(|state| state.running.unwrap())
```

Comment:
Hm... I'm a bit worried about these unwraps. Can we avoid them?

Reply:
`unwrap_or_default()` will make better sense here.